esp/services/ws_machine/metrics.xslt

```diff
@@ -520,11 +520,13 @@
                <table>
                   <tr>
                      <th id="selectAll1" colspan="5">
-                       <input type="checkbox" id="SelectAllCheckBox" title="{$hpccStrings/st[@id='SelectDeselectAll']}" onclick="selectAll0(this.checked)">
+                       <label><input type="checkbox" id="SelectAllCheckBox" title="{$hpccStrings/st[@id='SelectDeselectAll']}" onclick="selectAll0(this.checked)">
                          <xsl:if test="$selectAllChecked &gt; 0">
                            <xsl:attribute name="checked">true</xsl:attribute>
                          </xsl:if>
-                       <xsl:value-of select="$hpccStrings/st[@id='SelectAllOrNone']"/></input>
+                        </input>
```

Comment:
Should the input element be empty?

Reply:
Yes, it is empty. This is another way to label a checkbox:
 `<label><input type="checkbox" ... /> Label Text </label>`

Reply:
My bad. The transform content threw me off - I didn't realize it was modifying the attribute list.